media_utils/eme_puppeteer.py

```diff
@@ -0,0 +1,138 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from time import sleep
+import re
+from re import compile
+
+from marionette_driver import By, expected, Wait
+from marionette_driver.errors import TimeoutException, NoSuchElementException
+
+# Adapted from https://github.com/gavinsharp/aboutmedia/blob/master/chrome/content/aboutmedia.xhtml
+_debug_script = "var mainWindow = window.QueryInterface(Components.interfaces.nsIInterfaceRequestor)\
```

Comment:
If you use triple quotes around this string, you don't have to use continuations. Example: https://dxr.mozilla.org/mozilla-central/source/testing/marionette/driver/marionette_driver/marionette.py#792